Bio/SeqIO/SffIO.py

```diff
@@ -317,6 +291,8 @@ def _sff_file_header(handle):
     data = handle.read(31)
     if not data:
         raise ValueError("Empty file.")
+    elif isinstance(data, str):
```

Comment:
The pythonic way would be to call `struct.unpack(fmt, data)` below in a `try: except:` block. `struct.unpack` will raise a `TypeError` if `data` is not a bytes-like object. Then you don't need to use `isinstance` here.

Reply:
OK, that makes sense.

Reply:
Agreed.